__tests__/app.test.js

```diff
@@ -20,13 +33,16 @@ describe("GET /api/topics", () => {
         });
       });
   });
-  test("status: 404 - responds with with an error for any non existent endpoints", () => {
+});
+
+describe("GET /api", () => {
+  test("status: 200 - responds with with an array of topics, each with all endpoints available", () => {
     return request(app)
-      .get("/api/not-valid-endpoint")
-      .expect(404)
+      .get("/api")
+      .expect(200)
       .then(({ body }) => {
-        const { message } = body;
-        expect(message).toBe("endpoint not found");
+        expect(typeof body).toBe("object");
+        expect(body).toEqual(endpoints);
```

Comment:
This assertion implicitly checking if 'body' is an object as well, so could do on it's own here.